ipaserver/install/server/replicainstall.py

```diff
@@ -918,47 +918,55 @@ def install(installer):
 
 
 def ensure_enrolled(installer):
-    config = installer._config
+    # Prepare options for the installer script
+    args = [paths.IPA_CLIENT_INSTALL, "--no-ntp"]
```

Comment:
However, interactive mode will also allow the user to set domain/server before the confirmation if the IPADiscovery fails. Also, if IPADiscovery succeeds but the server/domain is not the one the user wants, they should be able to make their disapproval count.

This could be solved by having domain + server as mandatory options in ipa-replica-install but then we're losing the interactivity again.